app/model/events/EmailEvents.scala

```diff
@@ -31,4 +31,5 @@ object EmailEvents {
   case class ApplicationWithdrawn(to: String, name: String) extends EmailEvent
   case class AdjustmentsConfirmed(to: String, name: String, etrayAdjustments: String, videoAdjustments: String) extends EmailEvent
   case class AdjustmentsChanged(to: String, name: String, etrayAdjustments: String, videoAdjustments: String) extends EmailEvent
+  case class ApplicationExtendedToSdip(to: String, name: String) extends EmailEvent
```

Comment:
I think we have to choose here if we want to use converted or extended, or is it different?